content/blog/2020-08-18-dev-dittodb/index.md

```diff
@@ -0,0 +1,185 @@
+---
+slug: "dev-dittodb"
+title: Developing dittodb
+package_version: 0.1.1
```

Comment:
Add package version? I know you didn't show any code, but it's helps to pin things down if anything major changes in the future

Reply:
Is there something else that I need to add to make this show up? 0.1.1 is the current version for {dittodb}, I added it here following the technical guides, but I didn't see anything rendered with it on the post itself.

Reply:
Oh hmm. I thought 0.1.1 was the place holder. I'm not sure... @maelle, do you know how this works?

Reply:
If I remember correctly, I changed it from `0.1.0` to `0.1.1`, but regardless `0.1.1` is the actual real version that is released on CRAN. (it was `0.1.0` a bit ago, but was having issues with solars checks so had to bump the version 🙈)

Reply:
I don't even wanna talk about checks on CRAN now :sob: my weathercan package is struggling

Reply:
I think we'll leave this as good enough. Thanks!